docs/userstories/ClusterNetworkPolicy/Tenant_Namespace_Isolation.md

```diff
@@ -0,0 +1,28 @@
+### Summary
+
+As a cluster admin, I want to isolate all the tenants (modeled as Namespaces)
+on my cluster from each other by default.
+
+### Description
+
+Many enterprises are creating shared Kubernetes clusters that are managed by a
+centralized platform team. Each internal team that wants to run their workloads
+gets assigned a Namespace on the shared clusters. Naturally, the platform team
+will want to make sure that, by default, all intra-namespace traffic is allowed
```

Comment:
as discussed, we dont want to explicitly allow intra-namespace traffic, rather emphasize more on the deny of inter-namespace traffic

Reply:
Re-phrased